marco_site/assets/bower.json

```diff
@@ -9,7 +9,7 @@
     "ratchet": "~v1.0.0",
     "jquery.bbq": "master",
     "jqueryui-touch-punch": "*",
-    "bootstrap3-typeahead": "~3.1.0",
+    "bootstrap3-typeahead": "https://github.com/bassjobsen/Bootstrap-3-Typeahead.git",
```

Comment:
this will probably break at some point, probably want to target a specific tag or commit ID.